src/openassetio-core/include/openassetio/managerApi/ManagerInterface.hpp

```diff
@@ -166,7 +166,8 @@ class OPENASSETIO_CORE_EXPORT ManagerInterface {
    * Constructor.
    *
    * No work is done here - ManagerInterface instances should be cheap
-   * to construct and stateless.
+   * to construct. Any heavy-lifting should be done in @ref initialize,
+   * where relevant settings are also available.
```

Comment:
[Question] Does it matter if a user creates cheap local-state here? I think it dosen't, just checking that there isn't a rule that _all_ immutable state must be created in `initialize`

Reply:
Technically no reason they couldn't - the motivation here was to make things like [this](https://github.com/OpenAssetIO/OpenAssetIO/blob/d2fb418501d316a23ea84562590053ce50404397/src/openassetio-core/src/hostApi/ManagerFactory.cpp#L51) be suitably performant.